### DISCUSSION TITLE
Openweathermap shows incorrect air pressure

### DISCUSSION THREAD

**Discussion by Jojo1703** [mworion/MountWizzard4#327] [Feature Request] [answered]
Hello,

I have activated weather data from Openweathermap with a valid API code in MW4. Unfortunately, incorrect air pressure is displayed in the Environment window. In the debug I also see an incorrect specification of longitude and latitude. Here is the excerpt from Debug:

[2023-09-30 22:42:05.304][D][onlineWeather.p][ 258] http://api.openweathermap.org/data/2.5/forecast?lat=51.47&lon=0.00&APPID=d81e8ba30be0ef53f958a5228cc2fc21

I don't know where the incorrect entries of "lat" and "lon" come from. My correct data would be "lat=51.20" and "lon=7.36". Can I enter this somewhere and change it?

Bye
Jurgen

**Comment by mworion** [comment, marked as answer]
Hi Jürgen,

the Lat/lon is the value of your mount position. You could check this, when connected to your mount. Default is to Greenwich. If you are not connected you could enter the value there (which will be overwritten in case the mount will be connected. Master is mount position.
Please update to the latest release as you might not at v3.2.0.
If you are changing the location it still might take 12h to update as the data is cached. If you would change that, please stop MW4, delete the file meteoblue.data in the /data folder and restart again.

Hope this helps.
Michel 

![2023-10-01-11-03-21-MountWizzard4_-_v3 2 0](https://github.com/mworion/MountWizzard4/assets/6632980/981dd6a3-48cb-4cb1-8a29-16071a844044)

**Comment by Jojo1703** [comment]
Hello Michel,

Of course I entered the correct data for Lon + Lat into the mount, MW4 is updated to 3.2.1, I waited 24 hours, deleted the files openweathermap.data and meteoblue.data but I still see the wrong values ​​for air pressure and Temperature. NINA, for example, shows the correct values ​​with the same key. What else can I do?

greeting
Jurgen
![MW4_1](https://github.com/mworion/MountWizzard4/assets/65235013/07d0da00-802e-4cb9-98da-ce17babe79b6)
![MW4_2](https://github.com/mworion/MountWizzard4/assets/65235013/2c0304b4-efaa-43ba-9a79-09c8566cd06a)
![nina druck](https://github.com/mworion/MountWizzard4/assets/65235013/3ee67cae-ca0a-4b4d-9e84-7015065d3672)
[mw4-2023-10-02.log](https://github.com/mworion/MountWizzard4/files/12782800/mw4-2023-10-02.log)

**Comment by mworion** [comment]
Made formally an issue out of it
